video/vdu_buffered.h

```diff
@@ -12,6 +12,10 @@
 #include "multi_buffer_stream.h"
 #include "sprites.h"
 #include "types.h"
+#include "esp_heap_caps.h"
```

Comment:
as this file isn't included within our source code, I think this should be `#include <esp_heap_caps.h>` and sit above this list

Reply:
No problem. Easy fix. Again, you can do it, or I will, if you prefer.